[2023-07-17T09:13:50.604+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.data_acquisition manual__2023-07-17T09:13:22.241658+00:00 [queued]>
[2023-07-17T09:13:50.613+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.data_acquisition manual__2023-07-17T09:13:22.241658+00:00 [queued]>
[2023-07-17T09:13:50.613+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-17T09:13:50.631+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): data_acquisition> on 2023-07-17 09:13:22.241658+00:00
[2023-07-17T09:13:50.635+0000] {standard_task_runner.py:57} INFO - Started process 240 to run task
[2023-07-17T09:13:50.639+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'data_acquisition', 'manual__2023-07-17T09:13:22.241658+00:00', '--job-id', '646', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpb7d277w5']
[2023-07-17T09:13:50.642+0000] {standard_task_runner.py:85} INFO - Job 646: Subtask data_acquisition
[2023-07-17T09:13:50.689+0000] {task_command.py:410} INFO - Running <TaskInstance: main_dag.data_acquisition manual__2023-07-17T09:13:22.241658+00:00 [running]> on host a888360a1584
[2023-07-17T09:13:50.792+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='data_acquisition' AIRFLOW_CTX_EXECUTION_DATE='2023-07-17T09:13:22.241658+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-17T09:13:22.241658+00:00'
[2023-07-17T09:16:05.165+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/wikipedia/wikipedia.py:389 GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

The code that caused this warning is on line 389 of the file /home/***/.local/lib/python3.7/site-packages/wikipedia/wikipedia.py. To get rid of this warning, pass the additional argument 'features="lxml"' to the BeautifulSoup constructor.
[2023-07-17T09:16:05.178+0000] {logging_mixin.py:150} INFO - Could not load Casp
[2023-07-17T09:16:59.530+0000] {logging_mixin.py:150} INFO - Could not load Computer Vision
[2023-07-17T09:17:26.767+0000] {logging_mixin.py:150} INFO - Could not load Dbscan
[2023-07-17T09:17:34.755+0000] {logging_mixin.py:150} INFO - Could not load Data Mining
[2023-07-17T09:20:03.042+0000] {logging_mixin.py:150} INFO - Could not load Github
[2023-07-17T09:20:16.046+0000] {logging_mixin.py:150} INFO - Could not load Google
[2023-07-17T09:21:25.658+0000] {logging_mixin.py:150} INFO - Could not load Identity Matrix
[2023-07-17T09:23:13.260+0000] {logging_mixin.py:150} INFO - Could not load Machine Learning
[2023-07-17T09:24:05.065+0000] {logging_mixin.py:150} INFO - Could not load Multilayer Perceptron
[2023-07-17T09:24:12.944+0000] {logging_mixin.py:150} INFO - Could not load Np-Hard
[2023-07-17T09:25:31.781+0000] {logging_mixin.py:150} INFO - Could not load Pmc (Identifier)
[2023-07-17T09:25:40.998+0000] {logging_mixin.py:150} INFO - Could not load Petar Veličković
[2023-07-17T09:25:45.924+0000] {logging_mixin.py:150} INFO - Could not load Physics
[2023-07-17T09:25:53.188+0000] {logging_mixin.py:150} INFO - Could not load Pixel
[2023-07-17T09:26:58.700+0000] {logging_mixin.py:150} INFO - Could not load Relu
[2023-07-17T09:27:41.475+0000] {logging_mixin.py:150} INFO - Could not load Social Graph
[2023-07-17T09:28:08.354+0000] {logging_mixin.py:150} INFO - Could not load Tensorflow
[2023-07-17T09:28:09.874+0000] {logging_mixin.py:150} INFO - Could not load Thomas Kipf
[2023-07-17T09:28:44.842+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-07-17T09:28:44.853+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=data_acquisition, execution_date=20230717T091322, start_date=20230717T091350, end_date=20230717T092844
[2023-07-17T09:28:44.906+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-17T09:28:44.938+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
